src/screens/ContactScreen/index.js

```diff
@@ -106,17 +109,24 @@ const ContactScreen = ({navigation}) => {
   const handleCreateChannel = async () => {
     try {
       const mappingUserName = selectedUsers?.map((user) => user?.username).join(',');
+      const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
       let image = DEFAULT_PROFILE_PIC_PATH;
       if (selectedUsers.length === 1) {
         image = selectedUsers[0]?.profile_pic_path;
       }
       const dataSelected = {
         user: {
           name: mappingUserName,
-          image
+          image,
+          userId: mappingUserId
         }
       };
-      createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+
+      if (sourceScreen === ANONYMOUS) {
+        createAnonymousChat(dataSelected);
+      } else {
+        createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `mappingUserId` variable is declared to store the user IDs of the selected users. The `dataSelected` object now includes the `userId` field. A conditional statement is added to check if the source screen is `ANONYMOUS`, and if so, it calls `createAnonymousChat(dataSelected)`, otherwise it calls `createSignChat(followed, dataSelected, 'CONTACT_SCREEN')`. These changes allow for the creation of anonymous chats and include the user IDs in the chat data.
```diff
+ const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
+ let image = DEFAULT_PROFILE_PIC_PATH;
+ if (selectedUsers.length === 1) {
+   image = selectedUsers[0]?.profile_pic_path;
+ }
+ const dataSelected = {
+   user: {
+     name: mappingUserName,
+     image,
+     userId: mappingUserId
+   }
+ };
+ 
+ if (sourceScreen === ANONYMOUS) {
+   createAnonymousChat(dataSelected);
+ } else {
+   createSignChat(followed, dataSelected, 'CONTACT_SCREEN');
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->